src/zplghe_wrapper.c

```diff
@@ -92,15 +92,10 @@ dplasma_zplghe_operator( parsec_execution_stream_t *es,
  *
  ******************************************************************************/
 parsec_taskpool_t*
-dplasma_zplghe_New( double bump, dplasma_enum_t uplo,
+dplasma_zplghe_New( dplasma_enum_t uplo,
                     parsec_tiled_matrix_t *A,
-                    unsigned long long int seed)
+                    void *params)
 {
-    zplghe_args_t *params = (zplghe_args_t*)malloc(sizeof(zplghe_args_t));
-
-    params->bump  = bump;
```

Comment:
Suggested change:

Keep the same `dplasma_zplghe_New` prototype and the malloc here too, and do the free in `dplasma_zplghe_Destruct` you can get it from `tp->_g_op_args` if you cast `tp` to a `parsec_apply_taskpool_t *`